packages/plugin-hardhat/src/utils/hardhat-etherscan/resolveEtherscanApiKey.ts

```diff
@@ -0,0 +1,36 @@
+import { UpgradesError } from '@openzeppelin/upgrades-core';
```

Comment:
I assume other than the error type everything else in `utils/hardhat-etherscan` was vendored from `@nomiclabs/hardhat-etherscan`?

Let's put the [license file](https://github.com/NomicFoundation/hardhat/blob/%40nomiclabs/hardhat-etherscan%403.1.7/packages/hardhat-etherscan/LICENSE) in the directory.